test/bdd/fixtures/agent/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright IBM Corp, SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+version: '2'
+
+services:
+
+  alice.agent.example.com:
+    container_name: alice.aries.example.com
+    image: ${AGENT_FIXTURE_IMAGE}:${AGENT_FIXUTRE_IMAGE_TAG}
+    environment:
+      - GOPATH=/opt/gopath
+      - ID=${ALICE_AGENT_ID}
+      - API_HOST=${ALICE_HOST}:${ALICE_API_PORT}
+      - INBOUND_HOST=${ALICE_HOST}:${ALICE_INBOUND_PORT}
+      - WEBHOOK_URL=${ALICE_WEBHOOK_URL}
+      - DB_LOC=${ALICE_DB_LOC}
+    working_dir: /opt/gopath/src/github.com/hyperledger/aries-framework-go
```

Comment:
It is preferred to be outside the gopath, now that we use go modules.

Reply:
didn't quite get your comment.
This is directory is the one where command will run. Docker container will move to this directory once it starts.